datatypes/src/plots/multi_line_plot.rs

```diff
@@ -0,0 +1,110 @@
+}
+
+pub struct MultiLineChart {
+    data: Vec<DataPoint>,
+    measurement: Measurement,
+}
+
+impl MultiLineChart {
+    pub fn new(data: Vec<DataPoint>, measurement: Measurement) -> Self {
+        Self { data, measurement }
+    }
+}
+
+impl Plot for MultiLineChart {
+    fn to_vega_embeddable(&self, _allow_interactions: bool) -> Result<PlotData> {
+        let data = self
+            .data
+            .iter()
+            .map(|d| {
+                serde_json::json!({
+                    "x": d.time.as_rfc3339(),
+                    "y": d.value,
+                    "series": d.series,
+                })
+            })
+            .collect::<Vec<_>>();
+
+        let x_axis_label = "Time";
+        let y_axis_label = self.measurement.to_string();
+
+        let vega_string = serde_json::json!({
+            "$schema": "https://vega.github.io/schema/vega-lite/v4.17.0.json",
+            "data": {
+                "values": data
+            },
+            "description": "Multi Line Chart",
+            "encoding": {
+                "x": {
+                    "field": "x",
+                    "title": x_axis_label,
+                    "type": "temporal"
+                },
+                "y": {
+                    "field": "y",
+                    "title": y_axis_label,
+                    "type": "quantitative"
+                },
+                "color": {
+                    "field": "series",
+                    "scale": {
+                        "scheme": "category20"
+                    }
+                }
+            },
+            "mark": {
+                "type": "line",
+                "line": true,
+                "point": true
+            }
+        })
+        .to_string();
+
+        Ok(PlotData {
+            vega_string,
+            metadata: PlotMetaData::None,
+        })
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn serialization() {
+        let chart = MultiLineChart::new(
+            vec![
+                ("S0".to_owned(), TimeInstance::from_millis_unchecked(0), 0.).into(),
+                ("S1".to_owned(), TimeInstance::from_millis_unchecked(0), 2.).into(),
+                ("S0".to_owned(), TimeInstance::from_millis_unchecked(0), 1.).into(),
+            ],
+            Measurement::Unitless,
+        );
+        assert_eq!(
+            chart.to_vega_embeddable(false).unwrap(),
+            PlotData {
+                vega_string: r#"{"$schema":"https://vega.github.io/schema/vega-lite/v4.17.0.json","data":{"values":[{"x":"1970-01-01T00:00:00+00:00","y":0.0,"series":"S0"},{"x":"1970-01-01T00:00:00+00:00","y":2.0,"series":"S1"},{"x":"1970-01-01T00:00:00+00:00","y":1.0,"series":"S0"}]},"description":"Multi Line Chart","encoding":{"x":{"field":"x","title":"Time","type":"temporal"},"y":{"field":"y","title":"","type":"quantitative"},"color":{"field":"series","scale":{"scheme":"category20"}}},"mark":{"type":"line","line":true,"point":true}}"#.to_owned(),
```

Comment:
In this case, it shows something like Friday, 3rd as x label(s).
Is it possible to show it more accurately, does it make sense to do so?
I guess it behaves okay when you have more points.

Reply:
I'd too like to have the axis label exactly on the time of the data points. However we can't assume that the data is always regular like in the raster-value-extraction where all points have values at the exact same times. So we would need to have a special type of plots for this I think.